R/gg_diagnose.R

```diff
@@ -31,11 +30,16 @@
 #' @import ggplot2
 #' @importFrom gridExtra grid.arrange
 #' @importFrom stats fitted formula hatvalues qchisq qnorm quantile residuals rstandard
-gg_diagnose <- function(fitted.lm, theme = NULL, ncol = NA, plot.all = TRUE, 
+gg_diagnose <- function(fitted.lm, theme = NULL, ncol = NA, to.plot = "all", 
```

Comment:
We don't want to completely replace this argument. It'd introduce regression to people who're using this function with `plot.all` argument. We want a new argument instead.